<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ATB6XRestAssured"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.194">
    <properties></properties>

            <testcase
                name="validateResponseTest"
                time="0.194"
                classname="com.apiautomation.day_27June2024.gson.deserialization.Deserialization">

            <failure type="java.lang.NullPointerException" message="Cannot invoke &quot;com.apiautomation.day_27June2024.gson.serialization.BookingData.toString()&quot; because the return value of &quot;com.apiautomation.day_27June2024.gson.serialization.BookingResponse.getBookingData()&quot; is null">
                <![CDATA[java.lang.NullPointerException: Cannot invoke "com.apiautomation.day_27June2024.gson.serialization.BookingData.toString()" because the return value of "com.apiautomation.day_27June2024.gson.serialization.BookingResponse.getBookingData()" is null
                com.apiautomation.day_27June2024.gson.deserialization.Deserialization.validateResponseTest(Deserialization.java:28)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)java.base/java.lang.reflect.Method.invoke(Method.java:568)org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)java.base/java.util.ArrayList.forEach(ArrayList.java:1511)org.testng.TestRunner.privateRun(TestRunner.java:739)org.testng.TestRunner.run(TestRunner.java:614)org.testng.SuiteRunner.runTest(SuiteRunner.java:421)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)org.testng.SuiteRunner.run(SuiteRunner.java:312)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)org.testng.TestNG.runSuitesLocally(TestNG.java:1208)org.testng.TestNG.runSuites(TestNG.java:1112)org.testng.TestNG.run(TestNG.java:1079)com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
            </failure>
    </testcase>


</testsuite>
